package/Rvn.roc

```diff
@@ -572,6 +584,47 @@ decodeInt = \bytes, fromStr ->
         ['-', .. as digits] ->
             decodeUtf8Bytes bytes fromStr (1 + countUntil digits isDecimalDigit)
 
+        # Escaped single quoted char like '\n'
+        ['\'', '\\', char, '\'', ..] ->
```

Comment:
```suggestion
        ['\'', '\\', char, '\'', .. as rest] ->
```

I think if we add this `as rest` here to the remainder pattern, we can use `rest` instead of `List.dropFirst bytes 4` below. What do you think?